### PR TITLE
Feature networkrequsts api

### PR DESCRIPTION
**Pull request by jmartin777**
Why was it necessary?

- Created a new file for our Network Request method (fetch).
The "acquireMovieInfo" function is dynamic- it accepts a 
path parameter as an argument which allows you to make dynamic 
API calls by passing different paths to the function and retrieving 
different types of data by providing the appropriate path value.

[In our APP.js]

-The useState hook is used to manage state within a functional 
component, while the useEffect hook is used to perform side 
effects (in this case , our API calls) in response to component lifecycle events.

 The component initializes three states using the useState hook: 'movies' ,
'selectedmovie' and 'error'. The useEffect hook is used to fetch movies from 
the API when the component mounts. It calls the fetchMovies function which
uses the aquireMovieInfo to retrieve the movie data. This then updates the 
'movies' state with the fetched movie data or sets the 'error' state if the API call fails

How was it implemented? Describe the steps you took to implement the changes in this pull request.
Checklist:
[ ] Tests have been added or updated to cover changes
[x] Documentation has been updated (if necessary)
[x] Code follows project coding standards
[x] Changes have been reviewed by at least one other team member

### CODE REVIEW

**Comment by CapCinematic** [comment]
This is awesome, thanks for being very detailed in our use of hooks. This is helpful and another reason why I enjoy using hooks in general. I have the home button ready to go for our next push as well.

**Review by CapCinematic — COMMENTED** [review]
Looks well implemented, let's see if it breaks because I'm not sure if we need a method for page load.

File: src/APIcalls.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const acquireMovieInfo = (path) => {
+    return fetch(` https://rancid-tomatillos.herokuapp.com/api/v2/${path}`)
+    .then(response => {
+        if (!response.ok){
+            throw new Error("Failed to fetch movies")
```
Comment (hfaerber) [inline src/APIcalls.js:5]: Nice job checking for an ok response and throwing an Error

File: src/APIcalls.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const acquireMovieInfo = (path) => {
+    return fetch(` https://rancid-tomatillos.herokuapp.com/api/v2/${path}`)
```
Comment (hfaerber) [inline src/APIcalls.js:2]: Love the dynamic url path here!

File: src/App.js [hunk 3]
```diff
@@ -1,14 +1,31 @@
...
+    fetchMovies();
+  }, []);
+
+  const fetchMovies = () => {
+    acquireMovieInfo('movies')
+    .then((data) => {
+      setMovies(data.movies);
+    })
+    .catch((error) => {
+      setError(error.message)
```
Comment (hfaerber) [inline src/App.js:26]: Good use of your catch block to update the error state

**Review by hfaerber — COMMENTED** [review]
Nice work here!  I appreciate your thorough PR description. Keep it up!

**Pull request merged**